### PR TITLE
#625 Responsive Channel header and fix bug in ChannelIntermediate component

### PR DESCRIPTION
**Pull request by stephanevieira75**
# Preview:

Untitled_ Dec 4, 2020 12_27 PM (1)

## Mobile:
Iphone 5:
Capture d’écran de 2020-12-04 12-35-29

Iphone 6-7-8 plus:
Capture d’écran de 2020-12-04 12-35-38

## Tablet :
Ipad:
Capture d’écran de 2020-12-04 12-37-56

## Desktop:
Small screen: 
Capture d’écran de 2020-12-04 12-40-58

Large screen:
Capture d’écran de 2020-12-04 12-41-44

Xlarge screen:
Capture d’écran de 2020-12-04 12-42-20

### CODE REVIEW

File: twake/frontend/src/app/components/SearchPopup/SearchPopup.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default class SearchPopup extends React.Component {
       filterType: Search.type || false,
     };
 
-    this.search_enabled = (InitService.server_infos || {}).elastic_search_available !== false;
+    this.search_enabled = true;
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/components/SearchPopup/SearchPopup.js:33]: Can you just remove all usage of this variable instead?

File: twake/frontend/src/app/scenes/App.tsx [hunk 2]
```diff
@@ -22,9 +22,5 @@ export default () => {
     (Globals as any).store_publicAccess_get_data = WindowService.allGetParameter();
   }
 
-  return (
-    <BadDevice force={publicAccess}>
-      <ClientPage />
-    </BadDevice>
-  );
+  return <ClientPage />;
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/App.tsx:25]: Can you put again the "BadDevice" here?
We need it for old browsers too, so please keep it and just adapt the condition if you can :)
We don't support: IE and Opera mobile

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Small changes, thanks Stephane !

File: twake/frontend/src/app/components/SearchPopup/SearchPopup.js (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/components/SearchPopup/SearchPopup.js]: Done :+1:

File: twake/frontend/src/app/scenes/App.tsx (hunk 2 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/App.tsx]: Now minimum screen width is set to 300px

**Pull request merged**